.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: ParallelCluster CI
```

Comment:
nice! shall we remove travis config?

Reply:
From the repo settings in the github UI? I'm planning on doing so after merging these PRs.

Reply:
I was referring to the .travis.yml file

Reply:
Done in #287

Reply:
Also did this for the cookbook repo in [#910](https://github.com/aws/aws-parallelcluster-cookbook/pull/910)